src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1531,12 +1468,72 @@ paths:
                   $ref: '#/components/schemas/Group'
         default:
           $ref: '#/components/responses/defaultResponse'
+  /groups/{gid}/user/{uid}:
+    parameters:
+      - name: gid
```

Comment:
Please change parameter name to `groupId` to be consistent.

Reply:
Done